objectivec/GPBWellKnownTypes.h

```diff
@@ -49,4 +51,22 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithTimeIntervalSince1970:(NSTimeInterval)timeIntervalSince1970;
 @end
 
+// Extension to GPBAny to support packFrom and unpackTo for arbitrary messages.
+@interface GPBAny (GPBWellKnownTypes)
+// Initialize any with the given message. e.g., for google.protobuf.foo, type url
```

Comment:
Does it make sense to capitalize "any" so that it reads a little better, or quote it? I find it very difficult to read the documentation because I'm not sure if "any" is part of the sentence, or the type.